node_modules/oae-authentication/lib/strategies/google/init.js

```diff
@@ -42,13 +44,24 @@ module.exports = function() {
         // We fetch the config values *in* the getPassportStrategy so it can be re-configured at run-time.
         var key = AuthenticationConfig.getValue(tenant.alias, AuthenticationConstants.providers.GOOGLE, 'key');
         var secret = AuthenticationConfig.getValue(tenant.alias, AuthenticationConstants.providers.GOOGLE, 'secret');
+        var domains = AuthenticationConfig.getValue(tenant.alias, AuthenticationConstants.providers.GOOGLE, 'domains').split(',');
+
+        // Ensure we can do simple string comparisons by filtering empty domains and trimming out spaces
+        domains = _.chain(domains)
+            .map(function(domain) {
+                return domain.trim();
+            })
+            .filter(function(domain) {
+                return (domain && domain.length > 0);
+            })
```

Comment:
I haven't seen in what way we want to compare things yet, but should be lowercase in addition to trimming?